package.json

```diff
@@ -2,6 +2,7 @@
   "name": "coachco2",
   "version": "0.3.0",
   "scripts": {
+    "analyze": "COZY_SCRIPTS_ANALYZER=true yarn build",
```

Comment:
What is this? What does it do?

Reply:
ça lance l'analyseur de build afin de savoir combien pèse l'app, et qui pèse combien, parmi le vendors et les bundles

Reply:
Nice, TIL!